js/blocks/controls/multiselect.js

```diff
@@ -6,7 +6,7 @@ const BlockLabMultiselectControl = ( props, field, block ) => {
 
 	return (
 		<SelectControl
-			multiple="true"
+			multiple="multiple"
```

Comment:
This [document](https://wordpress.org/gutenberg/handbook/designers-developers/developers/components/select-control/#multiple) seems to call for a boolean value for `multiple`. But you've looked at this for much longer than me, so I'll defer to your judgement.

Reply:
There was a javascript Warning in the inspector suggesting to change it, due to the fact that although multiple="true" is valid, multiple="false" _isn't_, and could be confusing.